File: conection/lib/models/image.dart
class image{
  int id;
  String imageMotel;
  int motelId;

  image(int id, String imageMotel, int motelId){
    this.id = id;
    this.imageMotel = imageMotel;
    this.motelId = motelId;
  }

  image.fromJson(Map json)
      : id = json['id'],
        imageMotel = json['imageMotel'],
        motelId = json['motelId'];
}

File: Sourcode/back/Websitedangtintimkiemnhatro/Websitedangtintimkiemnhatro/Models/Image.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Websitedangtintimkiemnhatro.Models
{
    public class Image
    {
        [Key]
        public int Id { get; set; }
        public string ImageMotel { get; set; }
        public Motel Motel { get; set; }
        [ForeignKey("MotelId")]
        [Required]
        public int MotelId { get; set; }
    }
}
